src/test/java/org/broadinstitute/consent/http/db/DatasetDAOTest.java

```diff
@@ -936,6 +985,166 @@ void testGetApprovedDatasetsWhenEmpty() {
+    );
+    Integer voteId1 = voteDAO.insertVote(chairperson1.getUserId(), electionId1, VoteType.FINAL.getValue());
+    voteDAO.updateVote(true, "rationale", yesterday, voteId1, false, electionId1, yesterday, false);
+    electionDAO.updateElectionById(electionId1, ElectionStatus.CLOSED.getValue(), yesterday);
+    assertEquals(1, datasetDAO.getApprovedDatasets(user.getUserId()).size());
+
+    Integer electionId2 = electionDAO.insertElection(
+        ElectionType.DATA_ACCESS.getValue(),
+        ElectionStatus.OPEN.getValue(),
+        yesterday,
+        darCollection.getMostRecentDar().getReferenceId(),
+        dataset2.getDatasetId()
+    );
+    Integer voteId2 = voteDAO.insertVote(chairperson1.getUserId(), electionId2, VoteType.FINAL.getValue());
+    voteDAO.updateVote(true, "rationale", yesterday, voteId2, false, electionId1, yesterday, false);
+    electionDAO.updateElectionById(electionId2, ElectionStatus.CLOSED.getValue(), yesterday);
+    assertEquals(2, datasetDAO.getApprovedDatasets(user.getUserId()).size());
+
+    // Simulate 2 DAC 2 elections for today
+    Date today = new Date();
+    Integer electionId3 = electionDAO.insertElection(
+        ElectionType.DATA_ACCESS.getValue(),
+        ElectionStatus.OPEN.getValue(),
+        today,
+        darCollection.getMostRecentDar().getReferenceId(),
+        dataset3.getDatasetId()
+    );
+    Integer voteId3 = voteDAO.insertVote(chairperson2.getUserId(), electionId3, VoteType.FINAL.getValue());
+    voteDAO.updateVote(true, "rationale", today, voteId3, false, electionId3, today, false);
+    electionDAO.updateElectionById(electionId3, ElectionStatus.CLOSED.getValue(), today);
+    assertEquals(3, datasetDAO.getApprovedDatasets(user.getUserId()).size());
+
+    Integer electionId4 = electionDAO.insertElection(
+        ElectionType.DATA_ACCESS.getValue(),
+        ElectionStatus.OPEN.getValue(),
+        today,
+        darCollection.getMostRecentDar().getReferenceId(),
+        dataset4.getDatasetId()
+    );
+    Integer voteId4 = voteDAO.insertVote(chairperson2.getUserId(), electionId4, VoteType.FINAL.getValue());
+    voteDAO.updateVote(true, "rationale", today, voteId4, false, electionId4, today, false);
+    electionDAO.updateElectionById(electionId4, ElectionStatus.CLOSED.getValue(), today);
+
+    List<ApprovedDataset> approvedDatasets = datasetDAO.getApprovedDatasets(user.getUserId());
+    assertNotNull(approvedDatasets);
+    assertEquals(4, approvedDatasets.size());
+
+    // manually age the dar off
+    jdbi.useHandle(handle -> {
+      handle.createUpdate("UPDATE data_access_request SET submission_date=CURRENT_DATE - INTERVAL '365 days' WHERE data_access_request.reference_id = ?").bind(0, darCollection.getMostRecentDar().getReferenceId()).execute();
+    });
+
+    // confirm the expiration part of the query works.
+    List<ApprovedDataset> approvedDatasets2 = datasetDAO.getApprovedDatasets(user.getUserId());
+    assertEquals(0, approvedDatasets2.size());
```

Comment:
After the DAR is expired, ensure that we have no datasets approved for our user.